build/module-builder/Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM node:12-alpine as builder
 WORKDIR /botpress
-RUN apk add --no-cache git
-RUN git clone https://github.com/botpress/botpress.git
+COPY . botpress
```

Comment:
These lines make an interesting difference:
With git clone no Botpress context is reused - that makes issues in this PR easier to reproduce and might allow me to create my own builder image without manually cloning as a workaround until the official image is released.